docs/api.md

```diff
@@ -181,3 +181,18 @@ Example (generated using the body and public key from the previous examples):
     }
 }
 ```
+
+## Authenticated Media
+
+When accessing media from a Synapse homeserver with authenticated media enabled, an additional
+HTTP header needs to be passed along with any request to the Matrix Content Scanner.
+
+This header follows the Matrix specification `Authentication Bearer scheme` as outlined [here](https://spec.matrix.org/v1.12/client-server-api/#using-access-tokens).
```

Comment:
The Authentication Bearer scheme is credited to web standards, rather than the Matrix spec. We just use it.

```suggestion
This header follows the `Authentication Bearer scheme` as [outlined in the Matrix specification](https://spec.matrix.org/v1.12/client-server-api/#using-access-tokens).
```